edenai_apis/apis/google/google_text_api.py

```
@@ -447,6 +448,59 @@ def text__moderation(
             standardized_response=standardized_response,
         )
 
+    async def text__amoderation(
+        self, language: str, text: str, model: Optional[str] = None, **kwargs
```

Ruff (0.12.2)

Unused method argument: `model`

(ARG002)